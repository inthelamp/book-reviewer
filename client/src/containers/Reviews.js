import { Alert } from 'react-bootstrap'

/**
 * Reviews posted in public 
 */
const Reviews = () => {
    return (
        <Alert variant='info'>
            Under Construction
        </Alert>    
    )
}

export default Reviews
