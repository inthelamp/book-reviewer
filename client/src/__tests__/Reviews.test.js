import renderer from 'react-test-renderer'
import Reviews from '../containers/Reviews'

describe('Component: Reviews', () => {
    it('Component: Reviews rendering correctly', () => {  
        const userId = 'dfdfdf'
        const isSignedIn = true

        const tree = renderer
          .create(<Reviews userId={userId} isSignedIn={isSignedIn} />)
          .toJSON();
        expect(tree).toMatchSnapshot()
    })
})