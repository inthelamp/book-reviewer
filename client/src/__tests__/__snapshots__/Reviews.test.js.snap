// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Reviews Component: Reviews rendering correctly 1`] = `
<div
  className="review"
>
  <div>
    <h4>
      My Reviews
    </h4>
  </div>
  <h4>
    Reviews
  </h4>
</div>
`;
